{{cookiecutter.extension_name}}/.github/workflows/build.yml

```diff
@@ -28,8 +28,10 @@ jobs:
 {% if cookiecutter.has_server_extension.lower().startswith('y') %}
         pip install .
         jupyter lab build
-        jupyter serverextension list
-        jupyter labextension list
+        jupyter serverextension list 1>serverextensions 2>&1
+        cat serverextensions | grep {{ cookiecutter.extension_name|replace("-", "_") }}
```

Comment:
A better check would be to check for `OK` in the line, which will be true for server extension and lab extension:
e.g.:
`jupyter_server_proxy  OK` and `nbdime-jupyterlab v1.0.0  enabled   OK`

Reply:
I switch to regex "{{ cookiecutter.extension_name}}.*OK"